### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gjhannam**
Changes to spelling and grammar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,31 +175,34 @@ sql_config:
...
 Psycopg2 - https://pypi.org/project/psycopg2/ , http://initd.org/psycopg/
 ```
 pip install python-psycopg2
 ```
 
+_libpq-dev was required for linux, not windows_
 libpq-dev - https://pypi.org/project/libpq-dev/ , https://github.com/ncbi/python-libpq-dev
 ```
-pip install libpq-dev
+sudo apt-get install libpq-dev
```
Comment (ahaith) [inline README.md]: Was this supposed to change back to `sudo apt-get'?

Comment (gjhannam) [reply README.md]: Yes this time it was - i remembered I didn't need that for windows

Comment (ahaith) [reply README.md]: Ah OK, I see, it's only a unix requirement

File: README.md [hunk 2]
```diff
@@ -175,31 +175,34 @@ sql_config:
...
+
+2. Make sure Zegami CLI latest is installed 
 ```
 pip install zegami-cli[sql] --upgrade --no-cache-dir
 ```
 _Note: --no-cache-dir avoids some errors upon install_
 
-The login command promtps for username and password which is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the currenet users data directory.
-Once retrieved all subsequest commands will use the stored token, unless it is specifically overridden wiht the `--token` option
+Test the install with the login command, which promtps for username and password. This is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the currenet users data directory.
```
Comment (ahaith) [inline README.md]:
```suggestion
Test the install with the login command, which prompts for username and password. This is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the current users data directory.
```

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ pip3 install zegami-cli[sql]
 
 ## Login
 The login command promtps for username and password which is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the currenet users data directory.
```
Comment (ahaith) [inline README.md]:
```suggestion
The login command prompts for username and password which is then used to retrieve a long-lived API token which can be used for subsequent requests. The token is stored in a file in the current user's data directory.
```

**Pull request merged**